test/Raphael/raphael.test.ts

```diff
@@ -224,6 +224,41 @@ describe("Raphael DAO contract", () => {
                     expect(votesAgainst.eq(ethers.constants.Zero))
                 });
 
+                // tests vote()
+                it("getDidIVote returns correct values", async () => {
```

Comment:
same point as above with naming conventions

Reply:
changed to match your suggestion.